Modules/_ctypes/stgdict.c

```diff
@@ -306,7 +306,7 @@ PyCStructUnionType_update_stginfo(PyObject *type, PyObject *fields, int isStruct
     if (!tmp) {
         goto error;
     }
-    Py_ssize_t total_size = PyLong_AsInt(tmp);
+    Py_ssize_t total_size = PyLong_AsSsize_t(tmp);
```

Comment:
What was a reason for using `PyLong_AsInt()`. This is a new API, so it unlikely was accident. Is there a limitation on total_size to be in the range of `int`?

Reply:
It *was* an accident -- my mistake.